src/Lti/Exception/InvalidLtiVersionException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2020 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Lti\Exception;
+
+use RuntimeException;
+
+class InvalidLtiVersionException extends RuntimeException
+{
+
+}
```

Comment:
I've seen the usage of this class, but is there any sense of having an empty class, with a message provided from the place exception is thrown? Meaning that `InvalidLtiVersionException` doesn't anyhow extend RuntimeException. Because for now it doesn't differ from throwing `RuntimeException` with custom message.
Maybe it would make more sense to move the message inside the `InvalidLtiVersionException` class as the default one?